recipes/r-powereqtl/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set version = '0.3.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-powereqtl
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/powerEQTL_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/powerEQTL/powerEQTL_{{ version }}.tar.gz
+  sha256: fa870af84a5e68fec084714f045af4f8d1395863d6b67facfd29d4e2e4e49588
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-glmmadaptive
+    - r-nlme
+  run:
+    - r-base
+    - r-glmmadaptive
+    - r-nlme
+
+test:
+  commands:
+    - $R -e "library('powerEQTL')"           # [not win]
+    - "\"%R%\" -e \"library('powerEQTL')\""  # [win]
+
+about:
+  home: https://github.com/sterding/powerEQTL and https://bwhbioinfo.shinyapps.io/powerEQTL/
+  license: GPL-2.0-only
+  summary: 'Power and sample size calculation for bulk tissue and single-cell eQTL analysis based
+    on ANOVA, simple linear regression, or linear mixed effects model. It can also calculate
+    power/sample size for testing the association of a SNP to a continuous type phenotype.
+    Please see the reference: Dong X, Li X, Chang T-W, Scherzer CR, Weiss ST, Qiu W.
+    (2021) <doi:10.1093/bioinformatics/btab385>.'
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```

expand license (keeps parity with CRAN):

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```